sdk/src/azure/core/az_mqtt5_rpc_server.c

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/az_mqtt5_rpc_server.h>
+#include <azure/core/az_result.h>
+#include <azure/core/internal/az_log_internal.h>
+#include <stdlib.h>
+
+#include <azure/core/_az_cfg.h>
+
+AZ_NODISCARD az_mqtt5_rpc_server_options az_mqtt5_rpc_server_options_default()
+{
+  return (az_mqtt5_rpc_server_options){
+    .subscribe_timeout_in_seconds = AZ_MQTT5_RPC_SERVER_DEFAULT_TIMEOUT_SECONDS,
```

Comment:
In the RPC Client PR this will be simplified to 
```
/**
 * @brief The default timeout in seconds for subscribing/publishing.
 */
#define AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS 10
```
where it won't be sub specific, so I'll leave it out of the name - this is just a default and the application can set the subscribe and publish timeouts separately for the client (we don't have a timer on publishes for the server right now, but I'll add that to my list of things to think about for the future)